### PR TITLE
fix(charting): use red boxes only and for all duplicated category names, reorder CSS selectors to resolve styling conflicts, refine error message display to prevent overlapping with other elements and other messages PD-3121

### PR DESCRIPTION
**Pull request by CarlaCostea**
https://illuminate.atlassian.net/jira/software/c/projects/PD/issues/PD-3121?filter=myopenissues

### CODE REVIEW

**Comment by andreeapescar** [comment]
@CarlaCostea 
So you've merged all error messages together. Set makes sure you only have unique items. And you show the error message as if it was only for the first column?

I think one screenshot would've helped.

**Comment by CarlaCostea** [comment]
Hello. @andreeapescar  that description is accurate.
Here is a screenshot:
image

**Review by arimieandreea — APPROVED** [review]

File: packages/charting/src/axes.jsx [hunk 1]
```diff
@@ -182,13 +183,13 @@ export class TickComponent extends React.Component {
             barWidth={barWidth}
             rotate={rotate}
             correctness={correctness}
-            error={error}
+            error={error && (error[index] || error[index] == '')}
```
Comment (andreeimiron) [inline packages/charting/src/axes.jsx:186]: Do we need `error[index] == ''`?

Comment (CarlaCostea) [reply packages/charting/src/axes.jsx:186]: no, thank you, we do not need it anymore. I will remove it.

**Review by andreeimiron — APPROVED** [review]

**Pull request merged**